### ISSUE TITLE
Want to copy fields to tags automatically by post save signal method

### ISSUE THREAD

**Issue by engrimran7** [radiac/django-tagulous#170]
Hi i tried to copy fields like (title,model,brand,varient,size,) to the tags automatically instead of adding the same again manually 

i tried below code and it throughs an error which is already discussed here #41  but it dint helped at all.

 class resource(models.Model):
    title=models.CharField(max_length=100)
    size=models.CharField( max_length=20, default="")
    desc=models.TextField(default="")
    file=models.FileField(default="", blank=True)
    url= models.URLField(max_length=200, blank=True)
    Brand = models.ForeignKey(brand,on_delete=models.CASCADE, default="")
    Model = models.ForeignKey(model,on_delete=models.CASCADE, default="")
    Categories = models.ForeignKey(category,on_delete=models.CASCADE, default="")
    update_at=models.DateField(auto_now=True)
    slug=models.SlugField(max_length=200, null=True,blank=True)
    Tags = tagulous.models.TagField( to=Skill, help_text="This field does not split on spaces" )    
 
    def Tag(self):
        return ",".join([str(p) for p in self.Tags.all()] )
    def _str_(self):
     return self.title  
def tag_set(sender, instance,*arg, **kwargs):
    ans= array_tag(instance)
    mylist = ["firmware", "download", "gsm"]
    instance.Tags.add(mylist)  
post_save.connect(tag_set, sender=resource)

any quick fix will be highly appreciated

**Comment by radiac** [comment]
So just to confirm, you're getting the following error? 'list' object has no attribute 'pk'

**Comment by engrimran7** [comment]
yes i am getting the following error 'list' object has no attribute 'pk'

**Comment by engrimran7** [comment]
Any expert here please assist to resolve the issue if possible to resolve.
